lib/ueberauth/strategy/github.ex

```diff
@@ -234,18 +237,27 @@ defmodule Ueberauth.Strategy.Github do
     conn = put_private(conn, :github_token, token)
     # Will be better with Elixir 1.3 with/else
     case Ueberauth.Strategy.Github.OAuth.get(token, "/user") do
-      {:ok, %OAuth2.Response{status_code: 401, body: _body}} ->
```

Comment:
Are we fine with a generalized error here or do we actually want to keep the `Token Error` with `unauthorized` error when status code is 401? This could potentially be a breaking change if some consumers are depending on the `Token Error`. Let me know what you think?

Reply:
In production for me 401s are coming back as `{:error, %OAuth2.Response{status_code: 401, body: _body}}` (see my example in #57) which is why my app is blowing up here so I'm not even sure `{:ok, %OAuth2.Response{status_code: 401, body: _body}}` will match anything anymore. Maybe this has to do with my Oauth2 version? I tried to reproduce this with Github manually but I'm not even sure how to do that.

But if you want me to include it anyways I will...  Just let me know!

Reply:
Meh I added it back just in case.